### ISSUE TITLE
Consumer fails to read messages properly

### ISSUE THREAD

**Issue by fbricon** [jlandersen/vscode-kafka#23]
When running the [Quarkus Kafka Quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/master/kafka-quickstart#quarkus-kafka-quickstart), when a consumer is added to the prices topic, message encoding is borked (consumer expects UTF-8, producer simply [serializes integers](https://github.com/quarkusio/quarkus-quickstarts/blob/master/kafka-quickstart/src/main/resources/application.properties#L4) before sending to the topic):

<img width="358" alt="Screenshot 2020-11-23 at 11 30 37" src="https://user-images.githubusercontent.com/148698/99952452-fc2f8000-2d7f-11eb-8633-8a24e4e5700a.png">

**Comment by angelozerr** [comment]
I study quickly how to fix this issue, and I think we should use an avro js library to do that.

I tested quickly with https://www.npmjs.com/package/avsc and if key is a long we can write:

```ts
import { types } from "avsc";

message.key = new types.LongType().decode(message.key).value;
```
I wonder if we must use  https://github.com/apache/avro/tree/master/lang/js#avro-js instead?

Need some investigation...

**Comment by angelozerr** [comment]
> I wonder if we must use https://github.com/apache/avro/tree/master/lang/js#avro-js instead?

See https://github.com/mtth/avsc/issues/259 in other words, we must use `avsc`

**Comment by angelozerr** [comment]
This issue will be fixed with the issue https://github.com/jlandersen/vscode-kafka/issues/112

**Comment by fbricon** [comment]
Fixed with #112
